inc/azure_c_shared_utility/xlogging.h

```diff
@@ -109,6 +116,53 @@ typedef void(*LOGGER_LOG_GETLASTERROR)(const char* file, const char* func, int l
         } \
     } \
 }
+#else /*LOGERROR_CAPTURES_STACK_TRACES is defined*/ 
+extern char* getStackAsString(void);
+#define STACK_PRINT_FORMAT "\nStack:\n%s"
+#define LOG(log_category, log_options, format, ...)                                                                                                                      \
```

Comment:
>LOG(log [](start = 8, length = 7)

So according to an illustrious developer: what's the point if we don't do it for etw :-)?

Reply:
Now we do.

---
In reply to: [276822018](https://github.com/Azure/azure-c-shared-utility/pull/302#discussion_r276822018) [](ancestors = 276822018)